### ISSUE TITLE
`should be null` and `should not' (be null)` are failling

### ISSUE THREAD

**Issue by theburningmonk** [fsprojects/FsUnit#52]
Just tried these on 1.3.0.1 and both are throwing null ref exception
`"123" |> should be null`
`"123" |> should not' (be null)`
`null |> should be null`

using Nunit's `Assert.That(null, Is.Null)` works

**Comment by pver** [comment]
I had a similar issue with this until I noticed I forgot I had to use Null instead of null:

```
"123" |> should be Null
"123" |> should not' (be Null)
null |> should be Null
```

would be nice to have it working with 'null' as well though

**Comment by sergey-tihon** [comment]
Released in `v1.4.1`
